src/content/docs/dropins/payment-services/payment-dictionary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Payment Services Dictionary
+description: Reference the default dictionary for the Payment Services dropin.
+---
+
+import CodeImport from '@components/CodeImport.astro';
+import CodeInclude from '@components/CodeInclude.astro';
+
+The default Payment Services dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Payment Services drop-in.
```

Comment:
```suggestion
The default Payment Services dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Payment Services drop-in component.
```

Reply:
I think this is an exception to our branding guidance because our Payment Services product is branded as a proper noun (capitalized).